packages/hardhat-viem-assert/src/index.ts

```diff
@@ -0,0 +1,31 @@
+export function expect(fn: any) {
+  // TODO: function or value
+
+  return {
+    to: {
+      async changeEtherBalance(
+        publicClient: any,
+        address: any,
+        amount: bigint
+      ) {
+        const balanceBefore = await publicClient.getBalance({
```

Comment:
I believe we can get the public client from `hre.viem` instead of passing it as a parameter